src/redist/targets/GenerateLayout.targets

```diff
@@ -13,8 +13,9 @@
       <RoslynDirectory>$(OutputPath)/Roslyn</RoslynDirectory>
     </PropertyGroup>
     <ItemGroup>
-      <RoslynBits Include="$(NuGetPackagesDir)/microsoft.netcore.compilers/$(MicrosoftNETCoreCompilersPackageVersion)/tools/**/*" />
+      <RoslynBits Include="$(NuGetPackagesDir)/microsoft.net.compilers.toolset/$(MicrosoftNetCompilersToolsetPackageVersion)/tasks/netcoreapp2.1/**/*" />
     </ItemGroup>
+    <Error Condition="'@(RoslynBits)' == ''" Text="Something moved around in Roslyn package, adjust code here accordingly. TFM change?" />
```

Comment:
I tried wildcarding the netcoreapp2.1 part and it broke things as netcoreapp2.1 then got into recursivedir...

Reply:
I wonder if we all should stop putting a framework version number in our build task folders.

Reply:
Not a bad idea to just name `netcoreapp`

Reply:
It's not like it's a well-recognized name in NuGet anyway, it's just for organization